convergence.md

```diff
@@ -1,3 +1,501 @@
+* iojs-da
+* iojs-de
+* iojs-el
+* iojs-es
+* iojs-fa
+* iojs-fi
+* iojs-fr
+* iojs-he
+* iojs-hi
+* iojs-hu
+* iojs-id
+* iojs-it
+* iojs-ja
+* iojs-ka
+* iojs-ko
+* iojs-mk
+* iojs-ms
+* iojs-nl
+* iojs-no
+* iojs-pl
+* iojs-pt
+* iojs-ro
+* iojs-ru
+* iojs-sv
+* iojs-ta
+* iojs-tr
+* iojs-tw
+* iojs-uk
+* iojs-vi
+* platform-arm
+* platform-freebsd
+* platform-solaris
+* platform-windows
+* Streams
+* Supports
+* TC
+* Tracing
+* Website
+
+#### Repositories
+
+The current repositories within the io.js organization include:
+
+* io.js
+* roadmap
+* build
+* nan
+* website
+* evangelism
+* docker-iojs
+* readable-stream
+* LTS
+* hardware
+* tracing-wg
+* NG
+* node-addon-examples
+* doc-tool
+* build-containers
+* build-container-sync
+* logos
+* iojs-ro
+* iojs-bn
+* iojs-de
+* iojs-nl
+* iojs-da
+* iojs-mk
+* iojs-cs
+* iojs-sv
+* iojs-bg
+* iojs-hi
+* iojs-tw
+* iojs-ko
+* iojs-es
+* iojs-pl
+* iojs-id
+* iojs-vi
+* iojs-ja
+* iojs-tr
+* iojs-cn
+* iojs-ru
+* iojs-fr
+* iojs-hu
+* iojs-pt
+* iojs-uk
+* iojs-ka
+* iojs-fi
+* iojs-fa
+* iojs-ta
+* iojs-it
+* iojs-no
+* iojs-el
+* iojs-ms
+* iojs-ar
+* iojs-he
+
+#### iojs/io.js Branches and Tags:
+
+Branches:
+
+* master
+* mergev010/base
+* mergev010/cp-fixes
+* more-working-groups
+* next
+* old-master
+* only-consensus-seeking
+* roadmap
+* try
+* v0.10
+* v0.12
+* v1.8.0-commit
+* v1.x
+* vupdates
+* working-groups
+
+Tags:
+
+* v1.8.1
+* v1.7.1
+* v1.7.0
+* v1.6.4
+* v1.6.3
+* v1.6.2
+* v1.6.1
+* v1.6.0
+* v1.5.1
+* v1.5.0
+* v1.4.3
+* v1.4.2
+* v1.4.1
+* v1.3.0
+* v1.2.0
+* v1.1.0
+* v1.0.4
+* v1.0.3
+* v1.0.2-release
+* v1.0.2
+* v1.0.1-release
+* v1.0.1
+* v1.0.0-release
+* v1.0.0
+
+Within iojs/io.js, development is currently focused on `master` and `v1.x`.
+
+Current statistics show 11,369 commits (which include some overlap with joyent/node from the fork), 15 branches, 24 releases and 678 contributors.
+
+### Converging the Organizations
+
+1. Ownership of the current `github.org/iojs` organization will transfer to the Node.js Foundation.
+2. The Organization will be renamed from "io.js" to "Node.js Foundation". Note that renaming of the organization will cause the URL for the org to change. Github's infrastructure automatically handles the redirection. For the sake of this document, `github.com/nodejs-foundation` is assumed.
+3. The Owners of the Github Node.js Foundation Organization will include:
+  * The existing joyent node-coreteam members.
+  * The existing iojs TC members.
+  * Additional supporting staff selected by the Foundation Board.
+4. All members of the existing node-coreteam will be added as Collaborators within the organization
+5. A team representing the TSC members will be created (if one does not already exist). The membership of this team will be all current voting members of both the node-coreteam and io.js TC.
+6. Ownership of the following joyent/* repositories will be transferred to the Node.js Foundation organization:
+  * http://github.com/joyent/node
```

Comment:
I originally had it as `joyent-node` but there was some push back. This is an open discussion point tho. Whichever causes the least amount of disruption should be the preferred approach.